### PR TITLE
Add an option to disable popups for PS mixing txes

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

**Comment by troydash** [comment]
@UdjinM6 Thank you! ;)

File: src/qt/forms/optionsdialog.ui [hunk 1]
```diff
@@ -173,6 +173,16 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="showPrivatesendPopups">
```
[the remark this answers is not in the document]
Comment (UdjinM6) [reply src/qt/forms/optionsdialog.ui]: heh, fixed :)

**Comment by nmarley** [comment]
Nice! @UdjinM6 I'm assuming this addresses https://github.com/dashpay/dash/issues/613 ?

**Comment by UdjinM6** [comment]
@nmarley Ah, yes! Took 2 years to fix it 🙈 😄

**Pull request merged**